cfme/tests/webui/test_general_ui.py

```diff
@@ -465,6 +485,53 @@ def test_add_provider_button_accordion(has_no_providers, provider):
     assert view.add_button.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE)
+@pytest.mark.parametrize("option", ALL_OPTIONS)
+def test_infrastructure_provider_left_panel_titles(
+    setup_provider, provider, option, soft_assert
+):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/18h
+        testSteps:
+            1. Add an infrastructure provider and navigate to it's Details page.
+            2. Select Properties on the panel and check all items, whether they have their titles.
+            3. Select Relationships on the panel and check all items,
+                whether they have their titles.
+        expectedResults:
+            1.
+            2. Properties panel must have all items and clicking on each item should display
+                the correct page.
+            3. Relationships panel must have all items and clicking on each item should display
+                the correct page.
+    """
+    view = navigate_to(provider, "Details")
+    accordion = getattr(view.entities.sidebar, option)
+
+    for panel in ALL_OPTIONS[option]:
+        # Sometimes the panel doesn't enable immediately,
+        # using wait_for to give it some time to enable
+        wait_for(
+            lambda: not accordion.tree.is_disabled(partial_match(panel)),
+            fail_func=view.browser.refresh,
```

Comment:
Could you re-navigate rather than do a browser refresh?

Reply:
Why so?

Reply:
I think I've been told before that we should avoid browser refreshes in tests? @jawatts or @mshriver is this something we should generally avoid?

Reply:
Correct. It adds a lot of time to our test runs.

Reply:
Okay. I added `view.toolbar.reload.click` instead, I hope that works. Let me know if otherwise :)

Reply:
@john-dupuy @jawatts I cannot use `view.toolbar.reload.click` because it is not available on some pages. I'll have to go back to using `view.browser.refresh`, let me know if you have some other solution :)

Reply:
@valaparthvi another option is to force renavigation

Reply:
Wouldn't navigate take longer than page refresh?

Reply:
Agree and we wouldn't want to renavigate for a `fail_func`.

Reply:
So are we okay with `view.browser.refresh`?

Reply:
@valaparthvi yes, I am okay with it. Looking at the rest of the PR